README.md

```diff
@@ -1,53 +1,280 @@
+   - Enter the following parameters
+     - for **Scope name** use `access_as_user`
+     - Keep **Admins and users** for **Who can consent**
+     - in **Admin consent display name** type `Access ProfileAPI as a user`
+     - in **Admin consent description** type `Accesses the ProfileAPI Web API as a user`
+     - in **User consent display name** type `Access ProfileAPI as a user`
+     - in **User consent description** type `Accesses the ProfileAPI Web API as a user`
+     - Keep **State** as **Enabled**
+     - Select **Add scope**
+
+#### Register the client: ProfileSPA (in the same application registration)
+
+1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the configuration file for this project.
+1. From the app's Overview page, select the **Authentication** section.
+   - Click **Add a platform** button.
+   - Select **Web Applications** on the right blade.
+   - Add a **Redirect URIs**, for instance **http://localhost:3000**.
+   - Click **Configure**.
+1. Select the **API permissions** section
+   - Click the **Add a permission** button and then,
+   - Ensure that the **My APIs** tab is selected
+   - In the list of APIs, select the `ProfileAPIandSPA` API, or the name you entered for the Web API.
+   - In the **Delegated permissions** section, ensure that the right permissions are checked: **access_as_user**. Use the search box if necessary.
+   - Select the **Add permissions** button
+
+##### Configure the service app (ProfileAPI) to use your app registration
+
+>In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `ProfileAPI\appsettings.json` file
+1. Find the app key `Domain` and replace the existing value with your Azure AD tenant name.
+1. Find the app key `ClientId` and replace the existing value with the application ID (clientId) of the `ProfileAPIandSPA` application copied from the Azure portal.
+1. Find the app key `ClientSecret` and replace the existing value with the Client Secret of the `ProfileAPIandSPA` application copied from the Azure portal.
+
+##### Configure the client app (ProfileSPA) to use your app registration
+
+>In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `ProfileSPA\src\utils\authConfig.js` file
+1. Find the app key `clientId` and replace the existing value with the application ID (clientId) of the `ProfileAPIandSPA` application copied from the Azure portal.
+1. Find the app key `redirectUri` and replace the existing value with the base address of the ProfileSPA project (by default `http://localhost:3000/`).
+1. Find the app key `resourceUri` and replace the existing value with the base address of the ProfileAPI project (by default `https://localhost:44351/api/profile/`).
+1. Find the app key `resourceScope` and replace the existing value with *Scope* you created earlier `https://{tenant-domain}/{app-name}`.

```

Comment:
Find the app key `resourceScope` and replace the existing value with *Scope* you created earlier `https://{tenant-domain}/{app-name}/access_as_user`.

Reply:
good catch, thanks!